ui/app/templates/components/gutter-menu.hbs

```diff
@@ -68,9 +68,11 @@
         <li><LinkTo @route="topology" @activeClass="is-active" data-test-gutter-link="topology">Topology</LinkTo></li>
       </ul>
     </aside>
+    {{#if this.system.agent.version}}
```

Comment:
If we're type guarding `agent.config` in `system.js` just in case there's an API change where a case like `config` becomes `Config` (uppercasing because of how Go works then maybe we should add the Elvis operator here.

`this.system.agent?.version`

Reply:
This is also an implicit design decision to say if we don't have the `agent.version` then we won't render the Agent Version on the screen.

How do we approach this on the Hashicorp team?

Reply:
Ah... that's pretty odd. So we're not actually using JavaScript in `hbs` files. I didn't pick up on not having that option coming from React where we can use JavaScript in `jsx`.